android/src/main/java/io/mosip/tuvali/openid4vpble/Openid4vpBleModule.kt

```diff
@@ -236,6 +242,7 @@ class Openid4vpBleModule(private val reactContext: ReactApplicationContext) :
   // ) => EmitterSubscription;
 
   companion object {
+    var tuvaliVersion: String = "null"
```

Comment:
Default value can be "unknown" for better readability

Reply:
changed to unknown.